src/cryptonote_core/tx_pool.cpp

```diff
@@ -709,18 +740,19 @@ namespace cryptonote
     lock.commit();
   }
   //---------------------------------------------------------------------------------
-  size_t tx_memory_pool::get_transactions_count(bool include_unrelayed_txes) const
+  size_t tx_memory_pool::get_transactions_count(bool include_sensitive) const
   {
     CRITICAL_REGION_LOCAL(m_transactions_lock);
     CRITICAL_REGION_LOCAL1(m_blockchain);
-    return m_blockchain.get_txpool_tx_count(include_unrelayed_txes);
+    return m_blockchain.get_txpool_tx_count(include_sensitive);
   }
   //---------------------------------------------------------------------------------
-  void tx_memory_pool::get_transactions(std::vector<transaction>& txs, bool include_unrelayed_txes) const
+  void tx_memory_pool::get_transactions(std::vector<transaction>& txs, bool include_sensitive) const
```

Comment:
What is the semantic difference between "unrelayed" and "sensitive" here ? You might already have answered that somewhere, sorry :)

Reply:
`include_sensitive` now includes `do_not_relay`, `stem`, and any `local` tx  that originated at that node. The latter two are relayed immediately but still considered sensitive for the RPC calls. So this was a bit of bikeshedding and slight behavior change, but this is a source of potential leaks I think.